tests/testthat/test-run.R

```diff
@@ -41,7 +41,7 @@ test_that("get_params_summary returns a data.frame",{
 
   expected_summary <- dplyr::tibble(
     duration = 0.01, grid = TRUE, cpu_count = 1L,
-    output_type = "message", lib_path = "latest", did_error = FALSE
+    output_type = "message", case_version = 1L, lib_path = "latest", did_error = FALSE
```

Comment:
It might be nice to have a test where you exercise this explicitly.

In this case, I'm less worried about testing the behavior itself and more having an example to follow of how it should look | work.

Reply:
Yep, agreed. I want to clean this up a little and test for both the default case and the manually-set case

Reply:
Added tests